src/common/Finisher.cc

```diff
@@ -12,7 +12,7 @@
 void Finisher::start()
 {
   ldout(cct, 10) << __func__ << dendl;
-  finisher_thread.create();
+  finisher_thread.create("finisher");
```

Comment:
@yuyuyu101, I'll look into this, but If we do it here, we should also do it in src/common/WorkQueue.cc, because there is "thread_pool". @liewegas mention that work queue names are to long to fit in 15 buffer, but maybe we should do some shortcuts for them and put legend as a comment close to WorkQueue. Then this will be consistent, and thanks to pthread_setname_op() we will see what WQ is actually in thread_pool. What do you think?